### DISCUSSION TITLE
Please add OAuth provider for wechat

### DISCUSSION THREAD

**Discussion by jet10000** [fief-dev/fief#79] [Ideas]
https://wechat-oauth2.readthedocs.io/en/latest/index.html

**Comment by frankie567** [comment]
Hello @jet10000 👋 

Why not! OAuth providers implementations are provided by [`httpx-oauth`](https://github.com/frankie567/httpx-oauth), and it's quite easy to add a new one (you can draw inspiration from the [Discord](https://github.com/frankie567/httpx-oauth/blob/master/httpx_oauth/clients/discord.py) one, for example). I would be happy to review a PR and then enable it in Fief.
